staging/src/k8s.io/apiserver/pkg/util/wsstream/conn_test.go

```diff
@@ -59,7 +59,8 @@ func TestRawConn(t *testing.T) {
 		defer wg.Done()
 		data, err := ioutil.ReadAll(conn.channels[0])
 		if err != nil {
-			t.Fatal(err)
+			t.Errorf("%v", err)
```

Comment:
`t.Error(err)` would be sufficient here and in L146. Everything else looks good to me.

Reply:
Oh sorry, I could have been clearer: Meant that you can replace `t.Errorf("%v", err)` with `t.Error(err)`.